legend-sdlc-generation-file/pom.xml

```diff
@@ -128,6 +128,12 @@
             <artifactId>legend-engine-external-format-jsonSchema</artifactId>
             <scope>test</scope>
         </dependency>
+
+        <dependency>
+            <groupId>org.finos.legend.engine</groupId>
+            <artifactId>legend-engine-xt-graphQL-query</artifactId>
+            <scope>test</scope>
+        </dependency>
```

Comment:
Why is there a need to add this test dependency here? I don't see any tests or test data being updating in this module.

Reply:
Have removed it